Training_Center_Task_14/Tests.cs

```diff
@@ -0,0 +1,116 @@
+using NUnit.Framework.Interfaces;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Edge;
+using OpenQA.Selenium.Firefox;
+using OpenQA.Selenium.Remote;
+using OpenQA.Selenium.Support.UI;
+using System.Runtime;
+
+namespace Training_Center_Task_14
+{
+    public class Tests
+    {
+        private RemoteWebDriver _driver;
+
+        [SetUp]
+        public void Init()
+        {
+            String currentDate = DateTime.Now.ToString("yyyy-MM-dd");
+            String currentTime = DateTime.Now.ToString("yyyy-MM-dd HH:mm:ss");
+
+            var browserOptions = new ChromeOptions();
+            browserOptions.PlatformName = "macOS 12";
```

Comment:
please add configuration file (appsettings for exaple) to manage tests run

Reply:
Done.